bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -171,15 +128,16 @@ spec:
                 - bash
                 - -ec
                 - |
-                   if nodes=$(PGPASSWORD=${PGPOOL_POSTGRES_PASSWORD} psql -U {{ (include "postgresql-ha.postgresqlUsername" .) | quote }} {{- if not (empty (include "postgresql-ha.postgresqlDatabase" .)) }} -d {{ (include "postgresql-ha.postgresqlDatabase" .) | quote }}{{- end }} -h 127.0.0.1 -tA -c "show pool_nodes;" | grep "up" | wc -l); then
-                       if [[ $nodes -eq {{ $postgresqlReplicaCount }} ]]; then
-                           true
-                       else
-                           false
-                       fi
-                   else
-                       false
-                   fi
+                  if PGPASSWORD=${PGPOOL_POSTGRES_PASSWORD} psql -U {{ (include "postgresql-ha.postgresqlUsername" .) | quote }} {{- if not (empty (include "postgresql-ha.postgresqlDatabase" .)) }} -d {{ (include "postgresql-ha.postgresqlDatabase" .) | quote }}{{- end }} -h 127.0.0.1 -tA -c "show pool_nodes;"; then
```

Comment:
pgpool does not pay nicely with repmgr. if a `standby` node goes offline `pgpool` detect it and marks it as `down`. But when it comes back online `pgpool` does not update it's status. The only way to make `pgpool` to update it's status is to restart it. So in the liveness probe we are checking if any of the `down` nodes are back online and failing the liveness probe if thats the case, thereby triggering a pod restart